tools/influx_si/macros.xml

```diff
@@ -0,0 +1,295 @@
+                <param argument="--fseries" type="text" value="" label="--fseries" optional="true" help="File name with free parameter values for multiple starting points. Default: &#x27;&#x27; (empty, i.e. only one starting point from the FTBL file is used)" />
+                <param argument="--iseries" type="text" value="" label="--iseries" optional="true" help="Indexes of starting points to use. Format: &#x27;1:10&#x27; -- use only first ten starting points; &#x27;1,3&#x27; -- use the the first and third starting points; &#x27;1:10,15,91:100&#x27; -- a mix of both formats is allowed. Default: &#x27;&#x27; (empty, i.e. all provided starting points are used)" />
+                <param argument="--seed" type="integer" min="0" value="" label="--seed" optional="true" help="Integer (preferably a prime integer) used for reproducible random number generating. It makes reproducible random starting points (--irand) but also Monte-Carlo simulations for sensitivity analysis. Default: none, i.e. current system value is used, so random drawing will be varying at each run." />
+                <param argument="--excl_outliers" type="float" min="0" max="1" value="" label="--excl_outliers" optional="true" help="This option takes an optional argument, a p-value between 0 and 1 which is used to filter out measurement outliers. The filtering is based on Z statistics calculated on reduced residual distribution. Default: 0.01." />
+                <param argument="--tblimit" type="integer" min="0" value="0" label="--tblimit" optional="true" help="developer option: set trace back limit for python error messages" />
+                <param argument="--mtf" type="text" value="" label="--mtf MTF" optional="true" help="MTF is a coma separated list of files with following extensions: netw, linp, miso, mflux, mmet, tvar, cnstr, ftbl, vmtf. Only first 3 files are necessary to obtain a workable FTBL file, others are optional."/>
+            </section>
+        </inputs>
+    </xml>
+    <xml name="help">
+        <help><![CDATA[
+usage: influx_i [-h] [--noopt] [--noscale]
+        [--meth {BFGS,Nelder-Mead,nlsic,pso}] [--fullsys] [--emu]
+        [--irand] [--sens SENS] [--cupx CUPX] [--cupn CUPN]
+        [--cupp CUPP] [--clownr CLOWNR] [--cinout CINOUT]
+        [--clowp CLOWP] [--np NP] [--ln] [--sln] [--tikhreg] [--lim]
+        [--zc ZC] [--ffguess] [--fseries FSERIES] [--iseries ISERIES]
+        [--seed SEED] [--excl_outliers [EXCL_OUTLIERS]] [--nocalc]
+        [--addnoise] [--time_order {None,1,2,1,2}] [--version]
+        [--mtf MTF] [--prefix PREFIX] [--eprl EPRL] [--force FORCE]
+        [-o OUT] [--copy_doc] [--copy_test] [--install_rdep]
+        [--TIMEIT] [--prof] [--tblimit TBLIMIT]
+        [ftbl ...]
+
+Optimize free fluxes and optionally metabolite concentrations of a given static
+metabolic network defined in an FTBL file to fit 13C data provided in the same
+FTBL file.
+
+positional arguments:
```

Comment:
This should not be needed, as you explain the parameters above in every single param.